include/fastrtps/xmlparser/XMLProfileManager.h

```diff
@@ -168,11 +168,13 @@ class XMLProfileManager
             m_xml_files.clear();
             m_transport_profiles.clear();
 
+            /*
             for (auto pair : m_dynamictypes)
             {
                 types::DynamicTypeBuilderFactory::get_instance()->delete_builder(pair.second);
             }
             m_dynamictypes.clear();
+            */
```

Comment:
What's this and can it be removed if not needed?

Reply:
At the beginning of times, the DynamicTypesFactory wasn't responsible for the types created by the XMLParser. These dark ages are gone for a long time ago. It is safe to remove, so I'll remove it.